index.html

```diff
@@ -349,6 +349,23 @@ <h2>About Myself</h2>
 
 
 
+      </section>
+      <section class="contact-form">
+        <h2>Contact Me</h2>
+        <p>If you have an application you are interested in developing, a
+          feature that you need built or a project that needs coding.
+          I’d love to help with it!</p>
+        <form action="/contact" method="post" enctype="multipart/form-data">
+          <input type="text" placeholder="Full names"
```

Comment:
- [ ] From the project requirement, your name input should have `maxlength="30"` Kindly follow the project requirement.